docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,772 @@
+# Audit log filter functions, options and variables
+
+This feature is in [technical preview](../glossary.md#tech-preview).
+
+The following sections describe the [functions](#audit-log-filter-functions), [options, and variables](#audit-log-filter-options-and-variables) available in the audit log filter plugin.
+
+## Audit log filter functions
+
+The following audit log filter functions are available.
+
+| Function name |
+| --- | 
+| [audit_log_encryption_password_get(keyring_id)](#audit_log_encryption_password_getkeyring_id) |
+| [audit_log_encryption_password_set(new_password)](#audit_log_encryption_password_setnew_password) |
+| [audit_log_filter_flush()](#audit_log_filter_flush) |
+| [audit_log_read()](#audit_log_read) |
+| [audit_log_read_bookmark()](#audit_log_read_bookmark) |
+| [audit_log_filter_remove_filter(filter_name)](#audit_log_filter_remove_filterfilter_name) |
+| [audit_log_filter_remove_user(user_name)](#audit_log_filter_remove_useruser_name) |
+| [audit_log_rotate()](#audit_log_rotate) |
+| [audit_log_filter_set_filter(filter_name, definition)](#audit_log_filter_set_filterfilter_name-definition) |
+| [audit_log_filter_set_user(user_name, filter_name)](#audit_log_filter_set_useruser_name-filter_name)|
+
+
+
+### audit_log_encryption_password_get(keyring_id)
+
+This function returns the encryption password. Any keyring plugin or keyring component can be used, but the plugin or component must be enabled. If the plugin or component is not enabled, an error occurs.
+
+#### Parameters
+
+`keyring_id` - If the function does not contain a keyring_id, the function returns the current encryption password. You can also request a specific encryption password with the keyring ID of either the current password or an archived password.
+
+#### Returns
+
+This function returns either the current encryption password or a specific encryption password as a binary string.
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_encryption_password_get();
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +---------------------------------------------+
+    | audit_log_encryption_password_get()         |
+    +---------------------------------------------+
+    | password                                    |
+    +---------------------------------------------+
+    ```
+
+### audit_log_encryption_password_set(new_password)
+
+This function sets the encryption password and stores the new password in the keyring. 
+
+#### Parameters
+
+`password` - the password as a string. The maximum length is 766 bytes.
+
+#### Returns
+
+This function returns either a `1` for success or a `0` for failure.
```

Comment:
There should be a string like for the most of other functions

"A string that indicates whether the operation succeeded. OK indicates success. ERROR: message indicates failure."